app/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?><!-- Copyright (C) 2016 The Android Open Source Project
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<resources>
+    <!-- These are the options displayed in the gender drop-down Spinner -->
+    <string-array name="array_yes_no">
+        <item>@string/yes_string</item>
+        <item>@string/no_string</item>
+    </string-array>
+    <string-array name="array_breast">
+        <item>@string/left_breast_string</item>
+        <item>@string/right_breast_string</item>
+    </string-array>
+</resources>
```

Comment:
Don't forget the empty line at the end. You might need to configure the IDE to do it for XMLs too